src/main/resources/application.properties

```diff
@@ -26,4 +26,4 @@ E0/FAoGATJvuAfgy9uiKR7za7MigYVacE0u4aD1sF7v6D4AFqBOGquPQQhePSdz9\
 G/UUwySoo+AQ+rd2EPhyexjqXBhRGe+EDGFVFivaQzTT8/5bt/VddbTcw2IpmXYj\
 LW6V8BbcP5MRhd2JQSRh16nWwSQJ2BdpUZFwayEEQ6UcrMfqvA0=\
 -----END RSA PRIVATE KEY-----
-quarkus.openshift.route.expose=true
\ No newline at end of file
+quarkus.openshift.route.expose=true
```

Comment:
What was changed on this line @xstefank ?

Reply:
magic right? :D just a new line at the EOF as the default setting of my IDE. Don't worry about this.

Reply:
I'm not worried, but in Blame this line will be flagged as created by you in totally irrelevant commit instead of the commit that has to use this property   :)

Reply:
not worth repush, if there'll be something else, I'll fix this